tutorials/01_install.md

```diff
@@ -2,5 +2,75 @@
 
 Next Tutorial: \ref commandline
 
-Refer to [Ignition GUI's entry page](https://ignitionrobotics.org/libs/gui)
-for install instructions.
+[Install](#install)
+
+* [Binary Install](#binary-install)
+
+* [Source Install](#source-install)
+
+    * [Prerequisites](#prerequisites)
+
+    * [Building from Source](#building-from-source)
+
+# Install
+
+We recommend following the [Binary Install](#binary-install) instructions to get up and running as quickly and painlessly as possible.
+
+The [Source Install](#source-install) instructions should be used if you need the very latest software improvements, if you need to modify the code, or if you plan to make a contribution.
+
+## Binary Install
+
+On Ubuntu Bionic, it's possible to install Ignition GUI's version 1 as follows:
```

Comment:
Let's try not to mention specific versions so the tutorial will need less maintenance.

```suggestion
On Ubuntu Bionic, it's possible to install Ignition GUI as follows:
```